python/orca/dev/example/run-example-test-ray.sh

```diff
@@ -207,3 +237,4 @@ echo "#11 auto-xgboost-regressor-spark-df example time used:$time12 seconds"
 echo "#12 ray-dataset-xgboost example time used:$time13 seconds"
 echo "#13 orca brainMRI example time used:$time14 seconds"
 echo "#14 orca resnet50 inference example time used:$time15 seconds"
+echo "#15 mmcv faster_rcnn training example time used:$time16 seconds"
```

Comment:
Could you plz rebase the pr? I made some change to the test script last week. Align the numberings to make them beautiful! 😄 

Previous PR: https://github.com/intel-analytics/BigDL/pull/7851/files

Reply:
sure

Reply:
Contact @hamham223 to run this for the nightly build test; example test is now not included in the PR test to save time.